Add Japanese version and note about piano key re-rolls

katabame opened the pull request:
I added Japanese translate version of document, and wrote about piano key re-rolls.

katabame committed:
Add Japanese version

katabame committed:
Update Japanese translations

katabame committed:
Add note for piano keys

r-pufky commented on README.md:
```diff
@@ -55,7 +55,8 @@ the piano in Scarface's office and hit the following keys in order.
 ciphered riddles. These turn out to be required achievements to enable the
-secret.
+secret.  
+**Note**: Every time you hit the piano keys, these required achievments list are re-rolled.
```
```suggestion
**Note**: Every time you hit the piano keys, these required achievements list are re-rolled.
```

r-pufky commented on README_JA.md:
```diff
@@ -0,0 +1,479 @@
+Payday 2 シークレット
+---------------
+PD2のシークレットを解くための情報。
```
I'd add a note to refer to the english doc. This doc potentially can get out of date.

r-pufky reviewed (commented):
Awesome, thanks for the work!

fixed a spelling error, and requested that you leave a comment about this doc potentially getting out of date. Once those are resolved, I'll merge the pull request.

r-pufky committed:
Fix spell miss

Co-Authored-By: katabame <katabamia@gmail.com>

katabame committed:
Add language switcher

katabame committed:
Add language switcher

katabame committed:
Add attention for potentially can get out of date

katabame commented:
Thank you for the review, I fixed that spelling error and added language switcher. also added attention sentence for potentially getting out of date in Japanese version.
If I have to improve more, please notify me.🙌

r-pufky commented:
Really appreciate all the work you put into this!
